modules/cnv_configuring_masquerade.adoc

```diff
@@ -0,0 +1,35 @@
+[[configuring-masquerade]]
+=== Configuring masquerade mode
+
+You can use masquerade mode to hide a virtual machine's outgoing traffic behind
+the pod IP address. Masquerade mode uses Network Address Translation (NAT) to
+connect virtual machines to the pod network backend through a Linux bridge.
+
+Enable masquerade mode and allow traffic to enter the virtual machine by
+editing your virtual machine configuration file.
+
+.Prerequisites
+
+* The virtual machine must be configured to use DHCP to acquire IPv4 addresses.
+
+.Procedure
+
+. Edit the `interfaces` spec of your virtual machine configuration file:
++
+[source,yaml]
+----
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: red
+          masquerade: {} <1>
+          ports:
+            - port: 80 <2>
+  networks:
+  - name: red
+    pod: {}
+----
+<1> connect using masquerade mode
```

Comment:
Initial caps, please

Are both of these values static, or can you provide more information about how a user should set them?

Reply:
I'll fix the capitalization - oops. Thanks for pointing that out :)

I think those values are both static, but I'll try to find out more.

Thanks for the review, Kathryn!